graylog2-rest-client/src/main/java/org/graylog2/restclient/models/alerts/AlertCondition.java

```diff
@@ -116,6 +122,18 @@ public String getDescription() {
         return sb.toString();
     }
 
+    private String buildFieldStringValueDescription() {
```

Comment:
Useless use of `StringBuilder` in this method. Could be replaced by simple string concatenation which will also make it easier to read.